PCLumiProd/Corrections/corrPCC_2015_5TeV_v1.csv

```diff
@@ -0,0 +1,32 @@
+262157,1,500,0.9982
+262163,1,500,0.9981
+262165,1,500,0.9981
+262167,1,500,0.9978
+262168,1,500,0.9948
+262169,1,500,0.9990
+262173,1,500,0.9981
+262174,1,500,0.9978
+262204,1,500,0.9204
+262205,1,500,0.9186
+262235,1,500,0.9005
+262248,1,500,0.7080
```

Comment:
Lines like this should set off alarm bells.  It is probably a poor stats bin that we should throw out.